app/templates/index.html

```diff
@@ -1,18 +1,20 @@
+<!DOCTYPE html>
 <head>
     <title>KPS&#169;</title>
     <link rel="stylesheet" type="text/css" href="/static/css/style.css" />
+    <link rel="icon" href="/static/img/favicon.ico" type="image/x-icon"/>
 </head>
-
-<div class="container" style="overflow: hidden;">
-    <img id='photo' src="static/img/header.jpg" alt="Snow" style="height:100%;">
+<body>
+    <div class="container" onclick="loadPhoto()">
+        <img id='photo' src="static/img/header.jpg" alt="Snow"/>
```

Comment:
Mixing double and single quotes.
I wouldn't supply the `alt` tag at all, it's used to aid screenreaders, and the application is a digital photo frame so you can't really do that at all.